homeassistant/helpers/service.py

```diff
@@ -98,6 +98,8 @@ def extract_entity_ids(hass, service_call):
     """Helper method to extract a list of entity ids from a service call.
 
     Will convert group entity ids to the entity ids it represents.
+
+    This method must be run in the event loop.
```

Comment:
If it _must_ be run in the event loop, it should be renamed to start with `async_`. However, I think you mean here that it is async safe?